functorch/experimental/_cond.py

```diff
@@ -185,14 +179,15 @@ def _has_potential_branch_input_mutation(branch, fake_inputs):
 
     return False
 
-def _has_potential_branch_input_alias(branch, fake_inputs):
+
+def _has_potential_branch_input_alias(branch, inputs):
     """
     Dispatch-trace the branch with fake inputs and check if
     producing graph has output aliasing the branch input. This is
     bit restrictive as the branch must be traceable.
     """
     try:
-        gm = make_fx(branch)(*fake_inputs)
+        gm = make_fx(branch)(*inputs)
```

Comment:
nit: try to avoid unrelated refactor like this, it makes it harder for reviewer to see what's going on

Reply:
Refactored those fixes into https://github.com/pytorch/pytorch/pull/95988, so those changes should disappear in this PR after that one merges!